kentik_api_library/kentik_api/requests_payload/saved_filters_payload.py

```diff
@@ -24,7 +24,7 @@ class GetResponse:
     @classmethod
     def from_json(cls, json_string):
         dic = from_json(cls.__name__, json_string)
-        dic["company_id"] = convert(dic["company_id"], ID)
+        dic["id"] = convert(dic["id"], ID)
```

Comment:
does it need conversion? is the id in json a string? If so maybe GetResponse.id should also be str?

Reply:
Yes, I forgot to delete it. id does not need conversion here.